lib/utils/autocomplete.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const homedir = require('os').homedir();
+const readFileSync = require('./fs/readFileSync');
+const fileExistsSync = require('./fs/fileExistsSync');
+
+const tab = require('tabtab')({
+  name: 'serverless',
+});
+
+const autocomplete = () => {
+  const cacheFilePath = `${homedir}/.serverless/autocomplete.json`;
+  if (!fileExistsSync(cacheFilePath)) {
+    return;
```

Comment:
@brianneisler `serverless.init()` wouldn't work with the special `serverless completion` command because there's no plugin for it, that's why we're creating a cache file from the plugin manager after all. If we wanna do that we'd have to create a whole new plugin for that. Not worth it at all imo. Users just need to run any serverless command once to get that first cache file.